src/nvim/map_glyph_cache.c

```diff
@@ -0,0 +1,94 @@
+#include <nvim/api/private/helpers.h>
+#include <nvim/map.h>
```

Comment:
```suggestion
#include "nvim/api/private/helpers.h"
#include "nvim/map.h"
```
This file also doesn't have a PVS header.